auth-web/src/components/auth/staff/GLCodeDetailsModal.vue

```diff
@@ -1,185 +1,220 @@
+                    label="STOB (Standard Object of Expense)"
+                    v-model="glcodeDetails.serviceFeeStob"
+                  >
+                  </v-text-field>
+                </v-col>
+                <v-col cols="12" sm="6" class="col">
+                  <v-text-field
+                    filled
+                    hide-details
+                    label="Project Code"
+                    v-model="glcodeDetails.serviceFeeProjectCode"
+                  >
+                  </v-text-field>
+                </v-col>
+              </v-row>
+            </fieldset>
+
+            </div>
+
+            <v-divider class="mt-6 mb-8"></v-divider>
+
+            <div class="d-flex">
+              <v-spacer></v-spacer>
+              <v-btn
+                large
+                color="primary"
+                class="font-weight-bold mr-2"
+                @click="save"
+                width="100"
               >
-              </v-text-field>
-            </v-col>
-            <v-col cols="12" sm="4" offset-sm="4">
-              <v-text-field
-                filled
-                hide-details
-                label="Project Code"
-                v-model="glcodeDetails.serviceFeeProjectCode"
+                Save
+              </v-btn>
+              <v-btn
+                large
+                depressed
+                @click="close"
+                width="100"
               >
-              </v-text-field>
-            </v-col>
-          </v-row>
-          <v-row class="mb-10">
-            <v-col cols="12" sm="4">
-              <div class="gl-code-label">Associated Filing Types</div>
-            </v-col>
-            <v-col cols="12" sm="4">
-              <ul>
-                <li
-                  v-for="filing in filingTypes"
-                  :key="filing.feeScheduleId"
-                >{{filing.corpType}}/{{filing.filingType}}</li>
-              </ul>
-            </v-col>
-          </v-row>
-          <v-divider></v-divider>
-        </v-card-text>
-        <v-card-actions>
-          <v-spacer></v-spacer>
-          <v-btn
-            color="primary"
-            depressed
-            @click="save"
-            width="90"
-            class="font-weight-bold"
-          >Save</v-btn>
-          <v-btn
-            depressed
-            @click="close"
-            width="90"
-            class="font-weight-bold"
-          >Cancel</v-btn>
-        </v-card-actions>
-      </v-card>
-    </v-dialog>
-  </v-row>
+                Cancel
+              </v-btn>
+            </div>
+
+          </v-tab-item>
+          <v-tab-item>
+            <v-data-table
+              :headers="filingTypeHeaders"
+              :items="filingTypes"
+              item-key="feeScheduleId"
+              hide-default-footer
+            >
+              <template v-slot:loading>
+                Loading...
+              </template>
+              <template v-slot:item.corpTypeName="{ item }">
```

Comment:
We don't need the template here unless we need to format the column specifically.
just make sure the 'value' field in the 'filingTypeHeaders' has the correct variable name to map the data